modules/exploits/windows/local/bits_ntlm_token_impersonation.rb

```diff
@@ -152,11 +152,10 @@ def initialize(info = {})
   #   - Checks if the session is not already SYSTEM
   def check
     privs = client.sys.config.getprivs
-    os = client.sys.config.sysinfo['OS']
-
+    version = get_version_info
     # Fast fails
-    if os.include?('Windows 7') || os.include?('Windows XP')
-      print_bad("Operating system: #{os}")
+    if version.build_number < Msf::WindowsVersion::Win8 && !version.windows_server?
+      print_bad("Operating system: #{version.product_name}")
```

Comment:
Changed to include Windows 2000 and earlier, based on the comment below "Windows 7 and previous...not vulnerable"